src/BreweryScrapper.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2022B;
+
+class BreweryScrapper
+{
+    public const SHOW = "show";
+    public const LIST = "list";
+    public const CLEAR = "clear";
+    public const EXIT = "exit";
+
+    public array $data = [];
+
+    public function dataToArray(string $name): array
+    {
+        $choosenData = []; 
+        $handle = fopen(__DIR__ . "/../tests/stubs/table.csv", "r");
+        $headers = [];
+        $i = 0;
+
+        while (($row = fgetcsv($handle, null, ";")) !== false) {
+            if ($i === 0) {
```

Comment:
```suggestion
            if ($i++ === 0) {
```
Maybe this would be less problematic? With this approach you do not need to remember that you have to increment `$i` in every case.

Reply:
We can remove  the whole `if` statement and `$i` variable and assign to the `$headers` variable, first line from the file.
```php
$headers = fgetcsv($handle);
```
it will be simpler.